apps/src/lab2/views/components/Instructions/PredictQuestion.tsx

```diff
@@ -53,6 +58,16 @@ const PredictQuestion: React.FunctionComponent<PredictQuestionProps> = ({
     predictAnswerLocked &&
     !Boolean(predictResponse?.split(',').includes(index.toString()));
 
+  const onSubmitAnswer = () => {
+    dispatch(submitPredictResponse({appType: 'weblab2'}));
```

Comment:
could we make this a `useCallback` that takes in `appType` as a dependency? That way if another lab decides to also use the submit button flow we don't have to change this.

Reply:
Sure - sounds good!